recipes/wsgidav/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "WsgiDAV" %}
+{% set version = "3.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 415ac221806a266a3661f6f2f91ecc8dbcbe3653cdb1508abc0c397ea0cf35f3
+  patches:
+  - patches/0001-remove-setup_requires.patch
+  - patches/0002-use-cx_freeze-on-windows.patch  # [win]
+
+build:
+  # noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
The `pip install .` command looks alright, but somehow the package builds a wheel rather than installing from source.